Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+elifePipeline {
+    def commit
+    DockerImage image
+    stage 'Checkout', {
+        checkout scm
+        commit = elifeGitRevision()
+    }
+
+    node('containers-jenkins-plugin') {
+        stage 'Build images', {
+            checkout scm
+            dockerComposeBuild(commit)
```

Comment:
minor comment: by using the abstraction it's not quite clear to me what the command is doing exactly (e.g. that it might build both `docker-compose.yml` and `docker-compose.ci.yml`).

Reply:
All `dockerCompose*` Jenkins steps use both, by convention